src/Tribe/Main.php

```diff
@@ -3891,225 +3891,6 @@ public function isOrganizer( $postId = null ) {
-				if (
-					preg_match( $post_id_regex, $before, $post_id )
-					|| preg_match( $post_id_regex, $after, $post_id )
-				) {
-					$post_id = absint( $post_id[1] );
-				} else {
-					// if we can't find the post ID, then let's bail
-					return $where_sql;
-				}
-
-				// rebuild the WHERE clause
-				$where_sql = "{$before} (
-					{$known}
-					OR (
-						{$equal}
-						AND {$alias}.post_id {$gt_lt} {$post_id}
-					)
-				) {$after} ";
-			}
-
-			return $where_sql;
-		}
-
-		/**
-		 * Get the prev/next post for a given event. Ordered by start date instead of ID.
-		 *
-		 * @param WP_Post $post The post/event.
-		 * @param string  $mode Either 'next' or 'previous'.
-		 *
-		 * @return null|WP_Post
-		 */
-		public function get_closest_event( $post, $mode = 'next' ) {
-			global $wpdb;
-
-			if ( 'previous' === $mode ) {
-				$order      = 'DESC';
-				$direction  = '<';
-			} else {
-				$order      = 'ASC';
-				$direction  = '>';
-				$mode       = 'next';
-			}
-
-			$args = array(
-				'post__not_in'   => array( $post->ID ),
-				'order'          => $order,
-				'orderby'        => "TIMESTAMP( $wpdb->postmeta.meta_value ) ID",
-				'posts_per_page' => 1,
-				'meta_query'     => array(
-					array(
-						'key'     => '_EventStartDate',
-						'value'   => $post->EventStartDate,
-						'type'    => 'DATETIME',
-						'compare' => $direction,
-					),
-					array(
-						'key'     => '_EventHideFromUpcoming',
-						'compare' => 'NOT EXISTS',
-					),
-					'relation'    => 'AND',
-				),
-			);
-
-			/**
-			 * Allows the query arguments used when retrieving the next/previous event link
-			 * to be modified.
-			 *
-			 * @var array   $args
-			 * @var WP_Post $post
-			 */
-			$args = (array) apply_filters( "tribe_events_get_{$mode}_event_link", $args, $post );
-			add_filter( 'posts_where', array( $this, 'get_closest_event_where' ) );
-			$results = tribe_get_events( $args );
-			remove_filter( 'posts_where', array( $this, 'get_closest_event_where' ) );
-
-			$event = null;
-
-			// If we successfully located the next/prev event, we should have precisely one element in $results
-			if ( 1 === count( $results ) ) {
-				$event = current( $results );
-			}
-
-			/**
-			 * Affords an opportunity to modify the event used to generate the event link (typically for
-			 * the next or previous event in relation to $post).
-			 *
-			 * @var WP_Post $post
-			 * @var string  $mode (typically "previous" or "next")
-			 */
-			return apply_filters( 'tribe_events_get_closest_event', $event, $post, $mode );
-		}
-
-		/**
-		 * Get a "previous/next post" link for events. Ordered by start date instead of ID.
-		 *
-		 * @param WP_Post $post The post/event.
-		 * @param string  $mode Either 'next' or 'previous'.
-		 * @param mixed   $anchor
-		 *
-		 * @return string The link (with <a> tags).
-		 */
-		public function get_event_link( $post, $mode = 'next', $anchor = false ) {
```

Comment:
Don't remove the method, move them to the  Deprecated section and throw a `_deprecated_function()`